ui/shared/components/panel/LoadWorkspaceDataForm.jsx

```diff
@@ -125,11 +146,13 @@ const LoadWorkspaceDataForm = React.memo(({ params }) => (
       {WARNING_BANNER ? <Message error compact header={WARNING_HEADER} content={WARNING_BANNER} /> : null}
     </Segment>
     <FormWizard
-      onSubmit={createProjectFromWorkspace}
-      pages={FORM_WIZARD_PAGES}
+      {...props}
+      onSubmit={params.projectGuid ? addDataFromWorkspace(params.projectGuid) : createProjectFromWorkspace}
```

Comment:
since `params` is being set as the `initialValues`, `projectGuid` will be available in the `values` passed to the submit function so you don't need to bind it here